### ISSUE TITLE
Provide a create command that only provisions the DB / webhook site

### ISSUE THREAD

**Issue by mshick** [webhook/webhook#44]
Ideally I would be able to run some webhook provisioning command that allocates a site, and spits out info similar to what goes into the firebase.conf file, but doesn't do the whole directory build out. Kind of like `wh init` but for a site that doesn't exist yet.

To explain: I've created a slush generator for all my dev process, and ideally I'd be able to skip the whole initial site build which I'm not really using. I realize what I'm doing with the generator is outside your scope, but I think with this being a tool for devs trying to streamline their lives a little flexibility is nice. 

Even better would be a locally installable webhook / webhook api I can integrate more thoroughly, but I won't burden you with that request just yet. ;)

**Comment by snide** [comment]
I assume you're just using us a Firebase CMS? And again, assume the self-hosted version?

**Comment by mshick** [comment]
No, I'm making use of most of the webhook features. But for all my js / css / iconfont etc processes I'm using gulp. So, to roll up all that stuff with the webhook dirs I made a generator.

**Comment by LtSquigs** [comment]
This idea sounds good, but we probably wont be able to work on or implement something like this for a while, unfortunately just got other things we need to focus on right now.

However you can dig into the OS code for webhook and try to emulate what it does to create sites, hacky but a workable solution.
